onnxruntime/python/tools/quantization/base_quantizer.py

```diff
@@ -230,7 +231,16 @@ def quantize_bias_static_impl(self, bias_name, input_scale, weight_scale, beta=1
             # TODO: This formula should be explained including why the scale is not estimated for the bias as well.
             bias_scale = input_scale * weight_scale * beta
 
-            quantized_data = (np.asarray(bias_data) / bias_scale).round().astype(np.int32)
+            quantized_data = (np.asarray(bias_data) / bias_scale).round()
+            if (quantized_data < np.iinfo(np.int32).min).any() or (quantized_data > np.iinfo(np.int32).max).any():
```

Comment:
I would use logging instead of print so people can disable the message. Can you add numerical value about the scales? That would help users to understand what the problem is. Could you use f-strings as well?